tests/test_authentication.py

```diff
@@ -208,7 +209,10 @@ def test_with_too_new_jwt(self):
     def test_with_unknown_subject(self):
         auth = 'JWT ' + make_id_token(self.user.username + 'x')
         resp = self.client.get('/test/', HTTP_AUTHORIZATION=auth)
-        self.assertEqual(resp.status_code, 401)
+        if not api_settings.CREATE_USER:
+            self.assertEqual(resp.status_code, 401)
+        else:
+            self.assertEqual(resp.status_code, 200)
```

Comment:
It might be nicer to split this up into two tests, one that has the variable set to `True` and one to `False`. Because one of the test paths won't be reached this way.

Reply:
Thanks for your comments on testing.  Unfortunately, I'm not in a environment where I can test, or do any more work on oidc.  At the young age of 64, I was asked to retire, so I'm only doing a little coding now, e.g. aws lambda (zappa).  And I don't have access to an environment where I can continue to test and improve this type of authentication.  Perhaps you can continue to improve this.  Thanks for your comments